src/components/learner-credit-management/assignments-status-chips/FailedBadEmail.jsx

```diff
@@ -40,7 +40,7 @@ const FailedBadEmail = ({ learnerEmail }) => {
               </li>
               <li>
                 Get more troubleshooting help at{' '}
-                <Hyperlink destination="https://edx.org" target="_blank">
+                <Hyperlink destination="https://edx.org" showLaunchIcon target="_blank">
```

Comment:
nit: `showLaunchIcon` is the default, and could be removed (here and elsewhere):

<img width="934" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/7fec8e17-3e4a-40fb-8da5-5e07fbbf225a">